Kickstarter-iOS/Locales/de.lproj/Localizable.strings

```diff
@@ -71,11 +71,13 @@
 "Closes_project" = "Schließt das Projekt.";
 "Collapses_subcategories" = "Reduziert die Anzeige der Unterkategorien.";
 "Collections" = "Sammlungen";
-"Comment_reply_digest" = "Comment reply digest";
+"Comment_reply_digest" = "Tägliche Zusammenfassung Kommentare";
```

Comment:
The `Comment_reply_digest` was added by accident while running the `git add --patch` command